shoes-core/lib/shoes/dimension.rb

```diff
@@ -188,19 +178,13 @@ def valid_integer_string?(input)
     end
 
     def report_relative_to_parent_start
-      if element_start && parent && parent.element_start
-        element_start - parent.element_start
-      else
-        nil
-      end
+      start_exists = element_start && parent && parent.element_start
```

Comment:
instead of the local variable `start_exists` I'd rather have a little private method `start_exists?` - would make this even neater (imo) :)